.github/workflows/draft-pdf.yml

```diff
@@ -0,0 +1,23 @@
+on: [push]
```

Comment:
@eliotwrobson So we don't need to be uploading the paper on every single commit, do you think it's reasonable to limit this action to run only when files in the `joss` directory change? [This StackOverflow post](https://stackoverflow.com/questions/63822219/how-to-run-github-actions-workflow-only-if-the-pushed-files-are-in-a-specific-fo) indicates that GitHub Actions supports this out of the box with [path filters](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore).

I believe in our case, the syntax would be:

```yml
on:
  push:
    paths: ["joss/**"]
```

Reply:
Seems reasonable! will add.

Reply:
Done here: https://github.com/caleb531/automata/pull/163/commits/ef46c654e4d32e72519fe31bbf85a46121a0f737

Reply:
@eliotwrobson Good call for adding the `pull_request` section too. Closing this thread.